apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -41,6 +42,102 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     }
   }, [sections, selectedSectionId]);
 
+  const courseContent = (
+    <>
+      <Typography
+        semanticTag={'h2'}
+        visualAppearance={'overline-two'}
+        className={styles.sectionHeader}
+      >
+        {i18n.courseContent()}
+      </Typography>
+      <TeacherNavigationSidebarOption
+        icon={'desktop'}
+        optionTitle={i18n.course()}
+        isSelected={false}
+      />
+      <TeacherNavigationSidebarOption
```

Comment:
[TeacherDashboardPaths](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherDashboard/teacherNavigation/TeacherDashboardPaths.tsx) has a list of urls (which we will most likely eventually need to hook this up) and labels. Could we add the icon and title to that list and use it to populate these? Not sure if it's worth it, but it does seem like there's a bit of duplicate information in two different places.

Reply:
This is good to know!  I was actually just thinking of using something similar for managing the "selected" option.  I will keep this in mind for that PR - it seems like a good strategy!